hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -138,6 +139,10 @@
                      "+urg+lic+wait",
                      // Merge same assert instances found in different VDBs.
                      "-merge_across_libs",
+                     // This enables grading on the merged vdb.
+                     "-show tests",
+                     // Create a log file for good measure.
+                     "-log {cov_merge_dir}/merge.log",
```

Comment:
Is `cov_merge_report.log` that already gets created insufficient?

Reply:
Ohh, I just remembered - we delete the `cov_report.log` in dvsim. We should remove that section of code instead.

Reply:
Here it is: https://github.com/lowRISC/opentitan/blob/9cf8f4c75b144f4dbafcac7d42e30d42a9bd8742/util/dvsim/Deploy.py#L637

Reply:
I agree it is better to have the full log from make.

Done.